### ISSUE TITLE
Update for ACF v5?

### ISSUE THREAD

**Issue by bluestarstudios** [manuelbergeron/acf-file-dropzone#2]
Hi!

Do you plan on updating the plugin to work with ACF v5? I'd be happy to help. I actually went through and got most of the code integration on the PHP side working with ACF v5 coding standards while maintaining your v4 code as well for compatibility. but I'd love your help in getting the JS files updated to function properly.

Let me know if you'd be available to help with this.

Thanks!
David

**Comment by mecani** [comment]
Sorry I lost the credentials to this account. Fork it anytime you want!

**Comment by dxfan227** [comment]
@bluestarstudios  no idea how to get in touch in any other way, is this you ? https://codepen.io/bluestarstudios/pen/ZEGamYP

I have some questions.........if you are open whats the best way to get in touch?

**Comment by bluestarstudios** [comment]
@dxfan227 yes it is. If it's relating to the code pen, please comment in the thread on it and I'll do my best to help. Otherwise, you can find my email on my profile here. All the best. David
